scripts/setup-ubuntu.sh

```diff
@@ -137,7 +137,6 @@ PACKAGES+=" ruby"
 PACKAGES+=" libc-ares-dev"
 PACKAGES+=" libc-ares-dev:i386"
 PACKAGES+=" libicu-dev"
-PACKAGES+=" libicu-dev:i386"
```

Comment:
As far as I know nothing else in here uses libicu 32-bit, you may want to revert this if this is not the case